awsplugins/ec2/ec2.go

```diff
@@ -105,3 +105,12 @@ func getMetadata(imdsURL string, client *http.Client, token string) (*http.Respo
 
 	return client.Do(req)
 }
+
+// Metdata is deprecated.
+// It will be removed in the future but kept only for backward compatibility.
```

Comment:
Thanks! Can you go ahead and follow the convention here?

https://github.com/golang/go/wiki/Deprecated

Reply:
Oops, fixed by daa47e1